app/src/main/java/ee/ioc/phon/android/speechutils/AbstractAudioRecorder.java

```diff
@@ -40,25 +45,34 @@
     // 2 (bytes) * 1 (channels) * 30 (max rec time in seconds) * 44100 (times per second) = 2 646 000 bytes
     // but typically is:
     // 2 (bytes) * 1 (channels) * 20 (max rec time in seconds) * 16000 (times per second) = 640 000 bytes
-    private final byte[] mRecording;
+    final byte[] mRecording;
```

Comment:
visibility changed to package friendly due to inheriting class using this directly. This of course can be changed to a getter. It's a matter of taste I assume